lib/routes/account.js

```diff
@@ -960,7 +970,21 @@ module.exports = function (
         log.begin('Account.deviceDestroy', request)
         var sessionToken = request.auth.credentials
         var uid = sessionToken.uid
-        db.deleteDevice(uid, request.payload.id).then(reply, reply)
+        db.deleteDevice(uid, request.payload.id)
+          .then(reply, reply)
```

Comment:
So, this is embarrassing. I was just running through the steps to reproduce it again before adding them here and, quel surprise, no breakage. Whatever I thought it was to do with yesterday, evidently it wasn't. Or possibly I was imagining things.

Anyway, I'll push sanely-positioned `reply` calls to this branch and offer sincere apologies to @seanmonstar for any wild goose chases induced by my foolishness. It honestly seemed like an easily-repeatable issue yesterday... 😕